auth/client/src/test/java/org/wildfly/security/auth/client/DefaultSSLContextProviderFromCurrentAuthContextTestCase.java

```diff
@@ -0,0 +1,33 @@
+package org.wildfly.security.auth.client;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.wildfly.client.config.ConfigXMLParseException;
+
+import javax.net.ssl.SSLContext;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.GeneralSecurityException;
+import java.security.NoSuchAlgorithmException;
+import java.security.Security;
+
+public class DefaultSSLContextProviderFromCurrentAuthContextTestCase {
+    private static final String CONFIG_FILE = "file:./src/test/resources/org/wildfly/security/auth/client/test-wildfly-config-client-default-sslcontext.xml";
+
+    @Test
+    public void testDefaultSSLContextTakenFromCurrentContext() throws GeneralSecurityException, URISyntaxException, ConfigXMLParseException {
+        Security.insertProviderAt(new ElytronClientDefaultSSLContextProvider(), 1);
+        AuthenticationContext authenticationContext = ElytronXmlParser.parseAuthenticationClientConfiguration(new URI(CONFIG_FILE)).create();
```

Comment:
I'd add a Javadoc comment to this test method so that it is clear what it does, as you did in the previous ones.

BTW could you please convert the above mentioned (existing) comments to Javadoc as well?

Reply:
@fabiobrz I added javadoc on top of class in each test. I left in-line comments when I though it migh be useful to specify exact line. Thanks